overlord/ifacestate/ifacemgr.go

```diff
@@ -53,6 +54,8 @@ type InterfaceManager struct {
 // Manager returns a new InterfaceManager.
 // Extra interfaces can be provided for testing.
 func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.TaskRunner, extraInterfaces []interfaces.Interface, extraBackends []interfaces.SecurityBackend) (*InterfaceManager, error) {
+	perftimings := timings.New(map[string]string{"startup": "interface manager"})
```

Comment:
this should go after after the delayedCrossMgrInit

"startup": "ifacmgr"  ?  

s/perftimings/perfTimings/ ?